pkg/vulnsrc/ubuntu/ubuntu.go

```diff
@@ -213,3 +214,18 @@ func SeverityFromPriority(priority string) types.Severity {
 		return types.SeverityUnknown
 	}
 }
+
+// StatusFromUbuntuStatus normalises Ubuntu status into common Trivy Types
+func StatusFromUbuntuStatus(status string) types.Status {
+	switch status {
+	case "not-affected", "DNE", "not-vulnerable", "ignored":
```

Comment:
I don't think `not-affected`, `DNE` and `not-vulnerable` are "Will not fix". They're "Not affected", and we don't want to save them in the database as it increases the size.

Reply:
I agree.
I will push a change set for this today.

Reply:
`ignored` makes sense to me as it's considered "Will not fix".

Reply:
@knqyf263 I have pushed all the tidy up changes. Please re-review when you have a chance.
Thanks